include/dd/Package.hpp

```diff
@@ -237,6 +237,14 @@ namespace dd {
             return r;
         }
 
+        dEdge makeZeroDensityOperator() {
```

Comment:
this function is still missing a `QubitCount n` parameter similar to all the other related functions (`makeIdent`, `makeZeroState`)
technically, it is also missing the `std::size_t start = 0` parameter, but I am not so picky about that.

Reply:
Changed that.